### DISCUSSION TITLE
Release 4.1

### DISCUSSION THREAD

**Discussion by haugene** [haugene/docker-transmission-openvpn#2382] [Announcements]
TL;DR: This release bundles all changes to the master branch over the last year into version 4.1.

Hey everyone! :wave: 

I've been taking a break from this project for a while. Didn't plan for it to take this long, but life happened. Slowly getting back on the horse now and hope to not fall off it again for a while :crossed_fingers: :sweat_smile: 

A huge thanks to @pkishino :raised_hands: :bow: He has kept the project going the past year, and even before then he has been the number one contributor (myself included) for a while now.

Making releases has been my responsibility, and it's now over a year since the last one. High time to make some new ones. So here's the plan:

- Creating this 4.1 release as a marker as it's been too long already
- I'll merge the changes from the `dev` branch and include them in release 4.2
- Next up I think it's time to bump the Ubuntu base image to get a newer version of Transmission, that should be 4.3. But time will tell.

<hr /><em>This discussion was created from the release <a href='https://github.com/haugene/docker-transmission-openvpn/releases/tag/4.1'>Start releasing again</a>.</em>

**Comment by mkrupczak3** [comment]
Thanks for the free stuff! 🙏

Really like the tool, thanks for your time and energy maintaining 👨‍🏭

**Comment by SquareBottle** [comment]
Hey, welcome back!

Will this 4.1 release work on Synology DSM 7? If not, then how about 4.2?

**Reply by haugene** [reply]
Wasn't aware that 4.0 didn't work on Synology. Does the `dev` tag work? If so it should work with 4.2 which I'm in the process of releasing now.

**Reply by ilike2burnthing** [reply]
@haugene welcome back! Hope paternity leave treated you well 😉 

Synology's OS update to DSM 7 has broken quite a few applications (Sonarr jumps to mind), but there were wider issues going back to DSM 6 which weirdly only affected some users.

@SquareBottle there's a link in the tips & tricks doc to this 2yr old thread in case it's of any help and you haven't already seen it - https://github.com/haugene/docker-transmission-openvpn/issues/1542#issuecomment-793605649

**Reply by haugene** [reply]
Thanks @ilike2burnthing! It's been good, and it has been fun - but it can be exhausting :laughing: She has started to treat us better now and there's enough energy for hobbies again :see_no_evil:

**Reply by ilike2burnthing** [reply]
"Children are small but relentless opponents!" - Stephen Colbert

**Reply by SquareBottle** [reply]
I use Sonarr, and I didn't even notice when they were affected by the DSM 7 update. They must've updated Sonarr before I upgraded to DSM 7.

As for the tips and tricks in your link, they didn't work for me, but thanks anyway! I'll try a fresh install when 4.2 comes out since it sounds like that's when it'd make most sense to try again. Fingers crossed that current gen Synology devices will be supported again before the end of the year! (But hey, I'll be grateful _whenever_ it works again. I appreciate all your hard work, devs.)

**Reply by ilike2burnthing** [reply]
https://github.com/haugene/docker-transmission-openvpn/releases/tag/4.2

**Reply by ilike2burnthing** [reply]
In case you still have issues - https://github.com/haugene/docker-transmission-openvpn/discussions/2431
